extensions/functions/src/main/java/io/crate/window/NthValueFunctions.java

```diff
@@ -161,8 +162,8 @@ public static void register(ExtraFunctionsModule module) {
         module.register(
             Signature.window(
                 FIRST_VALUE_NAME,
-                parseTypeSignature("E"),
-                parseTypeSignature("E")
+                parse("E"),
+                parse("E")
```

Comment:
> Also renames the method name to encourage regular import
(TypeSignature.parse(...)) over static import.

Isn't this stating, we should prefer using `TypeSignature.parse()` instead of `parse()`? Just curious.

Reply:
Yes I was just to lazy to update all existing places, but I can change it

Reply:
👍 I don't mind.

Reply:
Changed them